skvlite/__init__.py

```diff
@@ -2,9 +2,10 @@
 import pickle
 import sqlite3
 from typing import Any, Generator, Mapping, Optional, Tuple, TypeVar, cast
-
+import zlib
```

Comment:
Please undo the formatting changes (and perhaps run `isort`) in order to make the `flake8` test pass.